pkg/version/windows_versioninfo.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Patch should be the `Z` in `X.Y.Z` -- right now we are not exposing it inside of the Bash library when we parse out the version but we can just add that.